pom.xml

```diff
@@ -2468,7 +2468,7 @@
                         <groupId>org.springframework.osgi</groupId>
                         <artifactId>spring-osgi-core</artifactId>
                     </exclusion>
-                    <!-- exlude this, its only used in the authentication plugin which we don't use -->
+                    <!-- exclude this, it's only used in the authentication plugin which we don't use -->
```

Comment:
How is that fixing a cve?

Reply:
My lazy for quick fix typo, I will split it in a separate commit

Reply:
I mean your commit description has nothing to do with what you do at the end :P

Reply:
Sorry obviously my comment wasn't clear... I do not see in your code an upgrade of `commons-configuration` from 2.7 to 2.8.0. You likely just fixed the typo, and forgot the lib upgrade :)